test/component_test.dart

```diff
@@ -50,8 +50,7 @@ void main() {
 
       // Cause store to trigger, wait for it to propagate
       store.trigger();
-      await nextTick();
-      expect(component.numberOfRedraws, 1);
+      await waitFor(() => component.numberOfRedraws == 1);
```

Comment:
i'd be inclined to just wait longer (add another `await nextTick()`) rather than rely on waiting for this numberOfRedraws to occur.  it's fine to ensure that this first redraw happens as expected, but it opens up the possibility that the next `store.trigger` (that happens just after this) isn't being tested properly (if this first trigger took longer than nextTick to render, then maybe then next trigger would too, and we're not waiting long enough to ensure that it didn't - can't use the waitFor method to ensure that it DIDN'T happen)